autofocus/predict/app/models/File.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from werkzeug import secure_filename
+
+
+class File:
```

Comment:
We don't need our own file class -- Python has built-in ways to do these things.

Reply:
Especially for our use case, I wrote the File class, so it would remove the file upon destructing the object. This removes the need to manually handle this every time while handling a file.

I think maybe the naming here is the problem, so I renamed this class to be 'TemporaryFile'.

I would like to keep this class, since the functionality here is pretty awesome and makes the handling very simple. However it is your call, if you don't see the value.

What do you think?

Reply:
Could we use https://docs.python.org/3/library/tempfile.html?